build/windows/installer/Package.wxs

```diff
@@ -0,0 +1,51 @@
+<Wix xmlns="http://wixtoolset.org/schemas/v4/wxs" xmlns:ui="http://wixtoolset.org/schemas/v4/wxs/ui">
+    <!-- ToDo use your own keys for Name and Manufacturer, as well as a new UUID for UpgradeCode -->
+    <Package
+        Name="BevyGame"
+        Manufacturer="My Name"
+        UpgradeCode="132bb59d-de4f-4893-a535-93dfd0597731"
+        Version="0.1.0"
```

Comment:
Is this the version that is used to control the update behaviour?

Reply:
Yep!

Reply:
Any chance to have the installer use the crate version for that? Otherwise I guess we should leave a comment here that this needs to be manually updated.